nbs/18_callback.fp16.ipynb

```diff
@@ -202,18 +202,49 @@
     "    def after_loss(self): self.autocast.__exit__(None, None, None)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.            loss_scale:int=512, # Loss scale to use if not using dynamic
</pre><p>Is there an easy explanation of what loss scale is or why it is 512 by default?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/muellerzr/fastai-docment-sprint/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
Good point, I don't really attempt to explain loss scale here. 512 is a fairly good default, I usually see dynamic settling at 256-512 for good values. Generally people shouldn't use this and instead use dynamic loss scale.

Reply:
so it should be a power of 2? or it can be any number like 17?

Reply:
Could be anything, but is generally a multiple of 2, because div_factor is by default set to be 2. Truthfully, I don't know anyone who would actually have a legitimate use case for actually using most of these settings, and I would expect them to instead simply use default dynamic scaling.